sdk/tests/sdk/whirlpools/quote/increase-liquidity-quote-by-liq.test.ts

```diff
@@ -0,0 +1,275 @@
+import { Percentage, ZERO } from "@orca-so/common-sdk";
+import assert from "assert";
+import BN from "bn.js";
+import {
+  PriceMath,
+  increaseLiquidityQuoteByLiquidityWithParams
+} from "../../../../src";
+import {
+  getTokenAFromLiquidity,
+  getTokenBFromLiquidity,
+} from "../../../../src/utils/position-util";
+
+const variations = [
+  [Percentage.fromFraction(0, 100), new BN(17733543)] as const,
+  [Percentage.fromFraction(1, 1000), new BN(17733543)] as const,
+  [Percentage.fromFraction(1, 100), new BN(17733543)] as const,
+];
+
```

Comment:
Generally same review comments for increase-liquidity-quote-by-input-token.test.ts 🙏